src/make-parameterized-roles.js

```diff
@@ -42,6 +42,7 @@ module.exports.run = async function(options) {
       'project:<..>:*',
       'queue:get-artifact:project/<..>/*',
       'queue:route:index.project.<..>.*',
+      'queue:create-task:low:project-<..>/*',
```

Comment:
Any reason this isn't `highest`?

Reply:
Oh, that's a good point -- I copied it from somewhere without thinking much.